blueocean-pipeline-api-impl/pom.xml

```diff
@@ -106,6 +106,13 @@
             <artifactId>plain-credentials</artifactId>
         </dependency>
 
+        <!-- Needed to read the "upstreamNode" action -->
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>pipeline-build-step</artifactId>
+            <version>2.7-SNAPSHOT</version>
```

Comment:
Version already defined under pluginManagement in parent pom. Not needed here.

Reply:
I thought so too, but it won't compile if I take this out.

Reply:
Hmm, this is strange. I will try it out locally.